doc/classes/TextEdit.xml

```diff
@@ -407,8 +421,10 @@
 			If [code]true[/code], line numbers are displayed to the left of the text.
 		</member>
 		<member name="smooth_scrolling" type="bool" setter="set_smooth_scroll_enable" getter="is_smooth_scroll_enabled">
+			If [code]true[/code], sets the [code]step[/code] of the scrollbars to [code]0.25[/code] which occurs in smoother scrolling.
```

Comment:
I'm not sure "occurs in" is correct here. "results in" is probably better, and easier to understand for non native speakers.